keyboards/acheron/austin/austin.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "quantum.h"
+
+#define KNO KC_NO
+
+#define LAYOUT_default( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, K0E, K0F, K10, K11, K12, \
+  K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, K1E, K1F, K20, K21, K22, K23, K24, K25, \
+  K26, K27, K28, K29, K2A, K2B, K2C, K2D, K2E, K2F, K30, K31, K32, K33, K34, K35, K36, K37, K38, \
+  K39, K3A, K3B, K3C, K3D, K3E, K3F, K40, K41, K42, K43, K44, K45, K46, K47, K48, K49, K4A, K4B, \
+  K4C, K4D, K4E, K4F, K50, K51, K52, K53, K54, K55, K56, K57, K58, K59, K5A, K5B, K5C, K5D, K5E, \
+  K5F, K60, K61, K62, K63, K64, K65, K66, K67, K68, K69, K6A, K6B, K6C, K6D, K6E, K6F, K70, K71  \
+) { \
+  {  K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, K0E, K0F, K10, K11, K12  }, \
+  {  K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, K1E, K1F, K20, K21, K22, K23, K24, K25  }, \
+  {  K26, K27, K28, K29, K2A, K2B, K2C, K2D, K2E, K2F, K30, K31, K32, K33, K34, K35, K36, K37, K38  }, \
+  {  K39, K3A, K3B, K3C, K3D, K3E, K3F, K40, K41, K42, K43, K44, K45, K46, K47, K48, K49, K4A, K4B  }, \
+  {  K4C, K4D, K4E, K4F, K50, K51, K52, K53, K54, K55, K56, K57, K58, K59, K5A, K5B, K5C, K5D, K5E  }, \
+  {  K5F, K60, K61, K62, K63, K64, K65, K66, K67, K68, K69, K6A, K6B, K6C, K6D, K6E, K6F, K70, K71  }, \
+}
```

Comment:
## Changeset A1

Please update Lines 7-21 to the following:

```c
#define LAYOUT_all( \
    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D,       K0E,  K0F, K0G, K0H, K0I, \
    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, K1E,  K1F,  K1G, K1H, K1I, K5I, \
    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2C, K2D,       K2E,  K2F, K2G, K2H, K3I, \
    K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C,            K3E,  K3F, K3G, K3H,      \
    K40, K41, K42, K43, K44, K45, K46, K47, K48, K49, K4A, K4B, K4C,   K4D,           K4F, K4G, K4H, K4I, \
    K50, K51, K52,                K55,                K59, K5A,   K5B, K5C, K5E,      K5F, K5G, K5H       \
) { \
    {  K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, K0E, K0F, K0G, K0H, K0I  }, \
    {  K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, K1E, K1F, K1G, K1H, K1I  }, \
    {  K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2C, K2D, K2E, K2F, K2G, K2H, KNO  }, \
    {  K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C, KNO, K3E, K3F, K3G, K3H, K3I  }, \
    {  K40, K41, K42, K43, K44, K45, K46, K47, K48, K49, K4A, K4B, K4C, K4D, KNO, K4F, K4G, K4H, K4I  }, \
    {  K50, K51, K52, KNO, KNO, K55, KNO, KNO, KNO, K59, K5A, K5B, K5C, KNO, K5E, K5F, K5G, K5H, K5I  }, \
}
```

This change:

- changes the name to `LAYOUT_all` [per QMK convention](https://docs.qmk.fm/#/hardware_keyboard_guidelines?id=ltkeyboard_namehgt) (I suspect this board may gain additional layout macros in future)
- renumbers all the switch locations to be `K<row><column>` by electrical matrix
- rearranges the layout macro such that the keycode positions in the keymap resemble the board's physical layout
- cleans up the line indents